resources/docker/images/authoring_local/tomcat/Dockerfile

```diff
@@ -56,7 +56,8 @@ RUN set -eux; \
     apk add --no-cache iproute2; \
     apk add --no-cache procps; \
 #    apk add --no-cache vim; \
-    apk add --no-cache nano;
+    apk add --no-cache nano; \
+    apk add --no-cache tini;
```

Comment:
I don't see `tini` used for this one, did I miss it?

Reply:
Updated